sample/src/main/AndroidManifest.xml

```diff
@@ -25,8 +25,20 @@
     xmlns:tools="http://schemas.android.com/tools">
 
     <application
-        android:label="Spark"
-        android:theme="@android:style/Theme.DeviceDefault"
-        tools:ignore="MissingApplicationIcon" />
+        android:label="@string/app_name"
+        android:theme="@style/SparkTheme.App"
+        tools:ignore="MissingApplicationIcon">
+
+        <activity
+            android:name=".MainActivity"
+            android:theme="@style/SparkTheme.App"
+            android:exported="true">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
```

Comment:
IMO we should not create this empty activity that only redirects to Showkase. Instead, make Showkase the launcher activity